### PR TITLE
Add some updates to support Shuup with Django 1.11 support

### PR DESCRIPTION
**Pull request by tulimaki**
Requires https://github.com/shuup/shuup/pull/1282

Since Shuup is yeat not updated here is proof that the tests pass:
screen shot 2018-06-15 at 4 06 33 pm

Refs shuup/shuup#1161

### CODE REVIEW

**Comment by chessbr** [comment]
@tulimaki can we fix this Travis build now that Shuup+Django 1.11 has been merged?

File: README.rst [hunk 1]
```diff
@@ -22,8 +22,12 @@ United States Of America
 License
 -------
 
-Shuup Wishlist is published under the GNU Affero General Public License,
-version 3 (AGPLv3). See the LICENSE file.
+Shuup YAML is published under Open Software License version 3.0 (OSL-3.0).
```
Comment (chessbr) [inline README.rst]: YAML ? :sweat_smile:

**Comment by tulimaki** [comment]
Fixed.

**Comment by tulimaki** [comment]
Fixed

**Pull request merged**